src/der.rs

```diff
@@ -14,16 +14,69 @@
 
 use crate::{calendar, time, Error};
 pub use ring::io::{
-    der::{nested, Tag, CONSTRUCTED},
+    der::{CONSTRUCTED, CONTEXT_SPECIFIC},
     Positive,
 };
 
+#[allow(clippy::upper_case_acronyms)]
+#[derive(Clone, Copy, Eq, PartialEq)]
+#[repr(u8)]
+pub enum Tag {
```

Comment:
nit: should this maybe be 
```suggestion
pub(crate) enum Tag {
```
since it's not publicly re-exported from the `der` module, and the `der` module is private?

Reply:
I followed the existing style of using `pub` and controlling exports in `src/lib.rs`. No strong preference though. If more reviewers think it's a good idea, I'll make the change.